### ISSUE TITLE
confidence output

### ISSUE THREAD

**Issue by bertsky** [qurator-spk/eynollah#135]
It would be great if Eynollah would annotate the confidence score of detected segments. From PAGE, this naturally can be represented via `Coords/@conf`. Of course, it would necessitate another refactoring, because this information gets cut short via np.argmax and not passed on...

**Comment by bertsky** [comment]
Has been implemented in https://github.com/qurator-spk/eynollah/commit/6b52da227c042c52d79ebbfc711e1a0f3b093e89 and https://github.com/qurator-spk/eynollah/commit/38a2d60fa2766aac3dc8f0412bb60315fa38ffdf on master – many thanks!

**Comment by bertsky** [comment]
Or should we keep the issue open until marginalia and non-light branches are extended for confidence aggregation?

(Or even other segments like TextLine, ImageRegion, SeparatorRegion, Border??)

**Comment by vahidrezanezhad** [comment]
> Or should we keep the issue open until marginalia and non-light branches are extended for confidence aggregation?
> 
> (Or even other segments like TextLine, ImageRegion, SeparatorRegion, Border??)

Yes, let's keep this issue open for now. Just for your information, 'marginalia' cannot have a confidence value since it is detected using heuristic methods. However, for all the other classes you mentioned, we can provide confidence values if needed — we should discuss this together later.

**Comment by bertsky** [comment]
> Just for your information, 'marginalia' cannot have a confidence value since it is detected using heuristic methods.

IIUC the base textline mask from the pixel classifier is also used for the text of marginals, though. So, even if the region as such is differentiated via rules, its pixel mass / substance could be aggregated over the matrix in the same way as the other region, couldn't it?
